packages/server/src/queue/queue.entity.ts

```diff
@@ -82,20 +79,10 @@ export class QueueModel extends BaseEntity {
     return open;
   }
 
-  @Expose()
-  get queueSize(): number {
```

Comment:
🚨 queueSize is no longer automatically sent to the client. **You must call `addQueueSize` if you want the queueSize present.**